README.md

```diff
@@ -110,11 +114,32 @@ Jump the current song to a time
 
 - milliseconds `Number` - Integer representing milliseconds to jump the current track to
 
+#### `playback.getSongInfo()`
+
+**Returns:**
+
+- song `Object` - Container for song info
+    - title `String` - Name of the song
+    - artist `String` - Artist of the song
+    - album `String` - Album of the song
+    - art `String` - URL for album art of the song
+    - duration `Number` - Milliseconds that the track will last for
+
 #### `playback.playPause()`
 Toggle between play and pause for the current song
 
 **This will not work if there are no songs in the queue.**
 
+### `playback.getPlaybackState()`
+
+**Returns:**
+
+- retVal `Number` - Phase that a song is in (e.g. 0, 1, 2)
```

Comment:
Yep, saw that when I got to the end of the README. I'm guessing I was tired of documenting every method and didn't want to take the time to think the proper word =/